templates/Error/error500.php

```diff
@@ -24,9 +24,9 @@
         <?= h($error->queryString) ?>
     </p>
 <?php endif; ?>
-<?php if (!empty($error->params)) : ?>
+<?php if (!empty($error->getBoundParams())) : ?>
```

Comment:
If $error can be multiple classes, should probably wrap all of this in a instanceof check.

Reply:
but the Error class also only has getBoundParams() and protected params ...

Reply:
Which exception class has `getBoundParams()`? I only see it on `Database\Statement\Statement`.

Reply:
This now needs to be a `method_exists()` check now.